src/app/auth/login/route.ts

```diff
@@ -0,0 +1,23 @@
+import { createRouteHandlerClient } from '@supabase/auth-helpers-nextjs'
+import { cookies } from 'next/headers'
+import { NextResponse } from 'next/server'
+import type { Database } from '@/app/auth/lib/database.types'
+
+
+export async function POST(request: Request) {
```

Comment:
same thing, in login/route.ts having one method with name POST doesn't looks solid, you can have better structure and better naming here.